examples/smartContracts/index.ts

```diff
@@ -211,6 +211,7 @@ const pollAsyncEvents = async (
         },
       ],
       web3Client,
+      fromMAS(0.1),
```

Comment:
@AurelienFT For a deploy sc, does the max_coins will always be equal to the **coins: fromMAS(0.1),** above

Reply:
I don't think so because it depends on the storage costs it could use and the more the SC is bigger the more it will needs coins

Reply:
ok so just to be sure, it's the sc builder responsibility to define max_coins